diox/pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: sentry_diox
+description: An integration which adds support for performance tracing for the Diox package.
+version: 6.19.0
+homepage: https://docs.sentry.io/platforms/dart/
+repository: https://github.com/getsentry/sentry-dart
+issue_tracker: https://github.com/getsentry/sentry-dart/issues
+
+environment:
+  sdk: '>=2.12.0 <3.0.0'
```

Comment:
diox requires Dart >=2.13.0 so there's no point in using Dart 2.12 as min here. Raising the min Dart version allows us to use newer features which we probably want to use if we need them.